tensilelite/Tensile/Common.py

```diff
@@ -700,6 +700,12 @@ def getArchitectureName(gfxName):
     # 1 indicates no assertion (since all sizes are multiples of 1)
     "AssertFree1ElementMultiple" : [1,2,4,8,16],
 
+    # Assertions that require arithmetic intensity to be specified value.
+    # Arithmetic intensity measures the ratio of computation to memory bandwidth required for a problem.
+    # These predicates can be used to adjust solution selection compute-bound or memory-bound problems.
+    "AssertAIGreaterThanEqual": -1,
+    "AssertAILessThanEqual":    -1,
+    
```

Comment:
Looks like this is missing the default value settings. There's a list further down in the file where defaults are defined.